cuda/install.bash

```diff
@@ -27,12 +27,11 @@ then
 else
     install_cuda=true
 fi
-
+return
```

Comment:
This seems to have popped up from no where

Reply:
It was for testing